src/types/types.js

```diff
@@ -6,7 +7,7 @@ export type TypeLayer = {
   y1: number,
   x2: number,
   y2: number,
-  options: {}
+  options?: { [string]: string }
```

Comment:
This type ain't right (the value can be all sorts of things). It'll be given a proper type in #86, but until then let's just give it an `any` type.

```js
options?: any
```

A useful trick for types like this is to setup a `TODO` type:

```js
type TODO = any
```

Which you can then use like this:

```js
options?: TODO
```